### DISCUSSION TITLE
Contribute to our Project idea and help beginner learn to code

### DISCUSSION THREAD

**Discussion by lostsoulremake** [community/community#66123] [Programming Help]
### Hello Everyone! ###
We have created a repository to help you contribute and make us provide great a good repo to beginners.
You can add links from Youtube / an course sites and help people learn with the project codes
https://github.com/Parthmagicss/Mern-Stack-Project-Ideas

Feel free to contribute, will surely merge the pull request, Do follow the following steps:
1. Fork the repo
[Redacted due to Code of Conduct]
2. Contribute in Readme.md
3. Push it to the main branch

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @lostsoulremake ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by queenofcorgis** [reply]
Additionally, I did remove a step in your post as  we currently do not allow self-promotion  on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
